### PR TITLE
Create ECD app

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: unicorecmsecd/locale/eng_GB/LC_MESSAGES/unicorecmsskeleton.po [hunk 1]
```diff
@@ -1,30 +0,0 @@
-#
```
Comment (smarzban) [inline unicorecmsecd/locale/eng_GB/LC_MESSAGES/unicorecmsskeleton.po]: @miltontony I removed the translations but I don't know how to create the new ones, as `django-admin` or `./manage.py makemessages` won't work here. how can I make the `.po` files 😕

**Comment by smarzban** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
@saeedmb8 you just need to run `./extract_i18n.sh` to generate translations.

**Comment by miltontony** [comment]
:+1: :cake: though

**Comment by miltontony** [comment]
:+1: this can land

**Pull request merged**